jsonnet/jsonnetfile.json

```diff
@@ -10,6 +10,16 @@
             },
             "version": "master"
         },
+        {
+            "name": "kube-thanos",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/metalmatze/kube-thanos",
```

Comment:
Didn't this move? cc @metalmatze

Reply:
Yes seems like it did https://github.com/thanos-io/kube-thanos @s-urbaniak can you change this as well.

Reply:
addressed, ptal :+1:

Reply:
oh just a second, i think i bumped the wrong file.